code/datums/quirks/positive_quirks/positive_quirks.dm

```diff
@@ -362,3 +362,36 @@
 
 /datum/quirk/item_quirk/signer/remove()
 	qdel(quirk_holder.GetComponent(/datum/component/sign_language))
+
+/datum/quirk/item_quirk/settler
+	name = "Settler"
+	desc = "You are from a lineage of the earliest space settlers! While your family's generational exposure to varying gravity \
+		has resulted in a...smaller height than is typical for your species, you make up for it by being much better at outdoorsmanship and \
```

Comment:
you use ellipses a lot in your writing style, are you sure they should be connected like x...y and not more like x... y? i see the second one most often but idk sources are confusing

Reply:
The proper use is actually x ... y but I just forget to do it properly and nobody has yet corrected me when i do it lol